oeps/processes/oep-0058-proc-translations-management.rst

```diff
@@ -0,0 +1,169 @@
+OEP-58: Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
+#################################################################################################################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-58 <oep-0058-proc-translations-management>`
+   * - Title
+     - Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
+   * - Last Modified
+     - 2022-08-08
+   * - Authors
+     - 
+       * Carlos Muniz <cmuniz@tcril.org>
+       * Feanil Patel <feanil@tcril.org>
+       * Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Ned Batchelder <ned@edx.org>
+   * - Status
+     - Accepted
+   * - Created
+     - 2022-08-08
+..    * - Resolution
+..      - 
+
+.. contents::
+  :local:
+  :depth: 1
+
+Context
+*******
+
+There are problems with the current method of managing translations via the
+edx-transifex-bot. The edx-transifex-bot uses a soon to be deprecated (Nov 30, 2022)
+version of the Transifex API, and requires admin rights on Transifex in order to
+function. In addition, it is difficult to track why some PRs are merged by the bot, and
+some are not, and where the bot is creating and merging PRs. Most recently, it was
+discovered that the translations were not uploading properly but it has been impossible
+to debug exactly why. In the week before the Nutmeg release, this was a significant pain
+point.
+
+Decision
+********
+
+To alleviate these issues, we will switch from using the edx-transifex-bot to the GitHub
+Transifex App, a stable app provided by Transifex. Benefits of this change include being
+easier to maintain and solving a lot of the pain points. As part of this proposal, we
+suggest moving translations into their own repository, to make using  the GitHub
+Transifex app more streamlined and straightforward, and in order to make organizing and
+using the up to date translations simpler.
+
+Description
+***********
+
+* Edx-transifex-bot is a potential security issue
```

Comment:
I'd work this into a sentence that's more complete and makes more sense immediately following a heading called "Description" - since this just kinda launches into problems. You might consider calling this section "Current State", describes(what we know!) about the current system, and only _then_ provides these bullet notes of problems (note at this point, your reader only has the context you provided in a 1 paragraph section above)

Reply:
I'll change the section to "Current State" as Description doesn't fit the content. In addition, I'll combine the sub-bullet point under the short one to add more detail in the first bullet point.